src/transformers/models/switch_transformers/configuration_switch_transformers.py

```diff
@@ -0,0 +1,206 @@
+# coding=utf-8
+# Copyright 2020, The SwitchTransformers Authors and HuggingFace Inc.
```

Comment:
Would that be Google given the original codebase google/flaxformers?

Reply:
.. and in 2f5806938d95a2ff819e240e18f0b50dd456edcb